config/locales/server.en.yml

```diff
@@ -24,11 +24,15 @@ en:
     experimental_inline_translation: "Enable experimental inline translation feature. This replaces existing parallel translation, allowing site visitors with a non-default locale to view content in their language."
     automatic_translation_target_languages: "The languages to automatically translate user content (posts, topics) to. If empty, no languages will be automatically translated."
   translator:
-    failed: "The translator is unable to translate this content (%{source_locale}) to the default language of this site (%{target_locale})."
+    failed: "The translator is unable to translate this %{content_type} (%{source_locale}) to your language (%{target_locale})."
```

Comment:
Interpolating translated strings like this could cause difficulty in accurate translations: I'm inclined to split these out into two complete strings, rather than inserting the translated `content_type` in the middle.

Reply:
Yah, the repeated text feels wasteful, but I've generally found it to be best to have the repeated text, then use the translation tools to help translators quickly copy and modify translations.

And if the translation does end up repetitive, `Content-encoding: gzip` means it only causes a few extra bytes to be downloaded by the end user. 😁